packages/modelviewer.dev/examples/scenegraph/index.html

```diff
@@ -499,6 +499,12 @@ <h2 class="demo-title">Transform textures</h2>
       <input type="range" min="0.5" max="1.5" value="1" step="0.01" id="scaleSlider">
       <p>Offset</p>
       <input type="range" min="0" max="1" value="0" step="0.01" id="offsetSlider">
+      <p>WrapMode</p>
+      <select id="wrapMode">
+        <option value="10497">Repeat</option>
```

Comment:
Can you switch this to `Wrap.Repeat` etc? I want to ensure our enum is properly exported.

Reply:
If it doesn't work, we probably have to re-export it from model-viewer.ts.

Reply:
We realized at the office that making this work might take a bit more time of work, leaving it for another index.html cleanup :/